src/Windows/Avalonia.Direct2D1/Media/Imaging/WicBitmapImpl.cs

```diff
@@ -53,17 +56,38 @@ public WicBitmapImpl(ImagingFactory factory, Stream stream)
         /// <param name="factory">The WIC imaging factory to use.</param>
         /// <param name="width">The width of the bitmap.</param>
         /// <param name="height">The height of the bitmap.</param>
-        public WicBitmapImpl(ImagingFactory factory, int width, int height)
+        /// <param name="pixelFormat">Pixel format</param>
+        public WicBitmapImpl(ImagingFactory factory, int width, int height, APixelFormat? pixelFormat = null)
```

Comment:
This is an enum so why not just do `APixelFormat pixelFormat = APixelFormat.Bgra8888`?